components/blocks/mdxComponentRenderer.tsx

```diff
@@ -172,7 +174,23 @@ export const componentRenderer: Components<{
     hideInternshipTestimonials: boolean;
   };
   EventLink: EventLinkProps;
+  ExpertBlock: {
+    header: string;
+    expertList: {
+      person: string;
+      profileImage: string;
+      peopleURL: string;
+      skills: string;
```

Comment:
this should be using a props object like the youtube playlist

Reply:
done